apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -509,30 +526,37 @@ var TopInstructions = React.createClass({
 
   render: function () {
     const resizerHeight = (this.props.collapsed ? 0 : RESIZER_HEIGHT);
-    const paddingToDeduct = this.props.hasContainedLevels ?
-      2 * CONTAINED_LEVEL_PADDING : 0;
+    const topInstructionsHeight = this.props.height - resizerHeight;
 
     const mainStyle = [
       this.props.isRtl ? styles.mainRtl : styles.main,
       {
-        height: this.props.height - resizerHeight - paddingToDeduct,
+        height: topInstructionsHeight,
       },
       this.props.isEmbedView && styles.embedView,
       this.props.noVisualization && styles.noViz,
       this.props.overlayVisible && styles.withOverlay,
-      this.props.hasContainedLevels && styles.containedLevel,
     ];
 
+
     if (this.props.hasContainedLevels) {
       return (
-        <div>
-          <div style={mainStyle} className="editor-column">
-            <ContainedLevel ref="containedLevel" />
+        <div style={mainStyle} className="editor-column">
+          <div
+            style={{
+              ...styles.containedLevelBackground,
+              height: topInstructionsHeight,
+            }}
+          >
+            <div style={styles.containedLevel} className="contained-level">
+              <ContainedLevel ref="containedLevel" />
+            </div>
           </div>
           {!this.props.collapsed && !this.props.isEmbedView &&
             <HeightResizer
               position={this.props.height}
               onResize={this.handleHeightResize}
+              style={styles.heightResizer}
```

Comment:
I added support for it in this PR. Is it bad practice to add a style property to react components?